### DISCUSSION TITLE
Additional document sorting options

### DISCUSSION THREAD

**Discussion by fuzyll** [outline/outline#5213] [Ideas]
I'd like to request some additional sorting options on top of the ones that were implemented to address #1370.

There are two edge-cases in particular that I've run into with the exiting sorting options:

1. Sometimes, I want to sort by reverse alphabetical order, and I don't see a way to actually do this? My specific example is that I like naming meeting notes as the date of the meeting, and I want the most recent meetings to be at the top of the list.

2. I would really like to be able to sort a set of child pages differently than the collection itself has been sorted. So, I might have a collection "Marketing" that contains a document "Meeting Notes" that contains sub-documents labeled by date. "Marketing" may have a custom sort option, while "Meeting Notes" may have a reverse alphabetical sort option.

For item 2, I would be perfectly fine with this being implemented as "sub-collections" (e.g. I have a collection inside of a collection, rather than a document with documents under it) if that would be easier. My specific use-case means that "Meeting Notes" document is _usually_ empty and has no content.

But, if it's not easier, having a document be able to change the sort order of sub-documents (and having more options for sorting) would be awesome.

EDIT: Sorry, thought of another one...

3. I would occasionally like to pin a couple of documents to the top of a collection. It would be nice to have sorting put those pinned documents above the others (and possibly also have some visual indication that they've been pinned). I currently do this by just naming things like "0 - Meeting Notes" so they bubble to the top, but it feels very clunky.

**Comment by tommoor** [comment]
These are reasonable asks, although for 1. "manual sort" is reverse date order by default so if you don't move docs around it can be used for your recent meetings at the top of the list case.

For 3 we already have pinned documents in collections that are displayed at the top, but not in the sidebar – I'm not sure we'd introduce another concept for this but perhaps a way to have existing pinned docs also top in the sidebar.

**Reply by fuzyll** [reply]
Yeah, I'm aware of the manual sort having reverse date order by default. I just unfortunately only want that for an "inner" collection of pages in my example. It works great otherwise!

I wasn't aware of already having a pinning thing available... I'd probably be fine with that being the mechanism for forcing things to the top of the sidebar, too.
